File: src/main/java/gauge/language/token/SpecTokenType.java
package gauge.language.token;

import com.intellij.psi.tree.IElementType;
import gauge.language.Specification;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public final class SpecTokenType extends IElementType {
  public SpecTokenType(@NotNull @NonNls String debugName) {
    super(debugName, Specification.INSTANCE);
  }

  @Override
  public String toString() {
    return "Specification." + super.toString();
  }
}

File: src/main/kotlin/gauge/language/token/SpecTokenType.kt
package gauge.language.token

import com.intellij.psi.tree.IElementType
import gauge.language.Specification

class SpecTokenType(debugName: String) : IElementType(debugName, Specification.INSTANCE) {

    override fun toString(): String {
        return "Specification.${super.toString()}"
    }
}
